builds/e2e/templates/nested-agent-deploy.yaml

```diff
@@ -45,19 +49,35 @@ steps:
       buildVersionToDownload: specific
       buildId: $(packageBuildId)
       downloadPath: '$(Build.StagingDirectory)'
-      artifactName: $(artifactName)                    
-  - task: CopyFiles@2
-    displayName: 'Copy Edgelet Artifact'
+      artifactName: $(artifactName)   
+  - task: PowerShell@2
+    displayName: 'Download aziot-identity-service'
     inputs:
-      SourceFolder: "$(Build.StagingDirectory)/$(artifactName)"
-      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/$(artifactName)"
-      CleanTargetFolder: true
-  - task: CopyFiles@2
-    displayName: 'Copy Images Artifact'
+      filePath: $(Build.SourcesDirectory)/scripts/local/test/DownloadIdentityService.ps1
+      workingDirectory: $(Build.SourcesDirectory)
+    env:
+      GITHUB_TOKEN: $(GitHubAccessToken)
+      ARTIFACT_NAME: $(identityServiceArtifactName)
+      PACKAGE_FILTER: $(identityServicePackageFilter)
+      DOWNLOAD_PATH: $(Build.StagingDirectory) 
+  - task: Bash@3     
+    displayName: 'Install artifacts'
```

Comment:
This is actually managed within E2E test framework. THe pipline shouldn't be the one dealing with this step :  https://github.com/Azure/iotedge/blob/master/test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManage
ment.cs

Reply:
The E2E framework doesn't run on lvl4/5